astroquery/mast/observations.py

```diff
@@ -908,6 +908,42 @@ def service_request_async(self, service, params, *, pagesize=None, page=None, **
 
         return self._portal_api_connection.service_request_async(service, params, pagesize, page, **kwargs)
 
+    def mast_query(self, service, columns='*', **kwargs):
+        # Specific keywords related to positional and MashupRequest parameters.
+        position_keys = ['ra', 'dec', 'radius', 'position']
+        request_keys = ['format', 'data', 'filename', 'timeout', 'clearcache',
+                        'removecache', 'removenullcolumns', 'page', 'pagesize']
+
+        # Explicit formatting for Mast's filtered services
+        if 'Filtered' in service:
```

Comment:
Currently it seems like the input `service` argument is case-sensitive, and I think people will have trouble with that if they don't capitalize the right letters. Can you make the `service` argument case-insensitive?